### DISCUSSION TITLE
Wunsch: Wartezeit EA Text

### DISCUSSION THREAD

**Discussion by NinavonReich** [Schrolli91/BOSWatch#532] [Ideas]
Randinfos:
Betrifft Multicast - Alarm
SW Version:	2.4.1
Branch:		dev
Build Date:	unrelease
___

Huhu Ihr! :)

Da ich jetzt länger nicht genervt habe, musste ich mir was aus den Finger sau... Nein, Spaß! :P

Ich habe - nur durch Zufall - festgestellt, dass etwa 4 % der Alarme nicht verarbeitet werden, da BOSWatch auf den EA - Text wartet und die alarmierte RIC dann irgendwann (zur eingestellten multicastAlarm_ignore_time passend) verwirft, wenn keiner folgt.

Da zu diesen Zeiten die Melder ganz normal mit Text ausgelöst haben, gehe ich von Empfangsproblemen aus und würde ich mich über eine neue Option freuen:

Ein maximaler Wartezeitraum auf die Text RIC, wie es ihn schon gibt, ABER:
Folgt diese nicht, dann wird dieser Alarm einfach ohne Text verarbeitet und nicht einfach gelöscht.

Liebe Grüße und schon vorab ein paar schöne weitere Adventstage! :)

**Comment by Schrolli91** [comment]
Kurzform:
Du willst eine Option, die nach dem Timeout den Alarm nicht verwirft, sondern einfach weiterreicht?
Soweit korrekt?

**Comment by NinavonReich** [comment]
Genau so. :)

**Comment by Kinick4** [comment]
Hallo! Das Feature würde ich mir auch wünschen. 
Eventuell könnte es auch als "erster Schritt" helfen, die Text-RIC aus dem Doppelungsfilter zu nehmen? Natürlich nur bei Multicast. Das würde höchstwahrscheinlich die Fälle abdecken können, bei denen die Text-Ric vor der Alarm-RIC decodiert wird, aber natürlich nicht die Fälle, wo keine Text-RIC empfangen wird...
Guten Rutsch euch!
